src/cdk/a11y/interactivity-checker.ts

```diff
@@ -9,11 +9,10 @@
 import {Injectable} from '@angular/core';
 import {Platform} from '@angular/cdk/platform';
 
-/**
- * The InteractivityChecker leans heavily on the ally.js accessibility utilities.
- * Methods like `isTabbable` are only covering specific edge-cases for the browsers which are
- * supported.
- */
+
+// The InteractivityChecker leans heavily on the ally.js accessibility utilities.
```

Comment:
`a11y.ts` instead of `.js`?

Reply:
With `ally.js` is a library meant, not a source file inside of Material. https://allyjs.io/